sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/NtfsFileAttributes.java

```diff
@@ -131,6 +131,7 @@ private static void toStringHelper(StringBuilder sb, EnumSet<NtfsFileAttributes>
         String[] splitAttributes = ntfsAttributes.split("\\|");
 
         for (String sa : splitAttributes) {
+            sa = sa.trim();
```

Comment:
Mind adding a test that directly targets this change.

Reply:
Thanks! I realized there werent any tests for those two methods and so I added more robust testing (also found a bug in fromAttributes so that's a win.)